keyboards/meetlab/rena/info.json

```diff
@@ -0,0 +1,116 @@
+{
+  "manufacturer": "meet_lab",
+  "keyboard_name": "Rena",
+  "maintainer": "yuezp",
+  "development_board": "bluepill",
```

Comment:
does this PCB require a STM32 Blue Pill development board?

Reply:
Shouldn't matter here.  It's using the same config (MCU, bootloader, HSE crystal, etc), so it's reasonable to use it here.

Reply:
Disagree. If we were going to build out `blackpill` as a converter for something like <https://mechwild.com/product/pillbug/>, it would cause similar issues to the current "everything is a promicro" mess.

While nothing directly stops you using the preset, it should really only be used where its actually using a development board.

Reply:
I didn't use a development board，Do I need to change it to the code?
`"processor": "STM32F103",
    "bootloader": "stm32duino",`

Reply:
> I didn't use a development board，Do I need to change it to the code?
"processor": "STM32F103", "bootloader": "stm32duino",

Correct, please make that change.

Reply:
> > I didn't use a development board，Do I need to change it to the code?
> > "processor": "STM32F103", "bootloader": "stm32duino",
> 
> Correct, please make that change.

thanks